sdk/core/azure-core-amqp/src/amqp/private/link_impl.hpp

```diff
@@ -79,6 +87,10 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
     std::shared_ptr<_detail::SessionImpl> m_session;
     Models::_internal::MessageSource m_source;
     Models::_internal::MessageTarget m_target;
+    OnLinkDetachEvent m_onLinkDetachEvent;
+    ON_LINK_DETACH_EVENT_SUBSCRIPTION_HANDLE m_linkSubscriptionHandle{};
+
+    static void OnLinkDetachEventFn(void* context, ERROR_HANDLE error);
 
 #if 0
```

Comment:
When I write #if 0, it is code for "this might be helpful in the future, but it's strictly not needed now".  In the case for link_impl, these are methods that exist in uAMQP but no in the LinkImpl class. I keep them here for reference so I can know what they are.

And if you have any suggestions for finding "commented-out code", I'd love to hear them.

Reply:
Searching for `//` in amqp, if > 1000, wearch for `//` with `(`?

As for disabled code, comment section above such code would help.